Jenkinsfile

```
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros':['centos7','sles12.3','leap42.3'],
-                         'rpmlint_check': false])
+packageBuildingPipelineDAOS(['distros':['centos8','centos7'],
```

This actually shouln't use this method.  There is no value in the build-DAOS test for this project.
```suggestion
packageBuildingPipeline(['distros':['centos8','centos7'],
```

---

https://github.com/daos-stack/pipeline-lib/pull/88